main.tf

```diff
@@ -139,3 +67,70 @@ module "monitoring" {
   web_test_availability_request_url = module.load_balancer.lb_dns_name
   measure_latency                   = var.monitoring_route53_measure_latency
 }
+
+module "graphdb" {
+  source               = "./modules/graphdb"
```

Comment:
Please use consistent ordering of the properties along the modules. 
e.g., 

```
module "alabala" {
  source = ""
  <empty line>
  count = ""
  <empty line>
  aws_region = ""
  ..
  ..
}
```

Reply:
Fixed.